pennylane/pauli/dla/lie_closure.py

```diff
@@ -49,6 +51,10 @@ def lie_closure(
             This can help with performance to avoid unnecessary conversions to :class:`~pennylane.operation.Operator`
             and vice versa. Default is ``False``.
         tol (float): Numerical tolerance for the linear independence check used in :class:`~.PauliVSpace`.
+        dtype: Dtype for the underlyding matrix representation in :class:`~.PauliVSpace`.
+            For generators that are pure Pauli words (:class:`~PauliWord`), it can be set
+            to ``float``. For sums of Paulis (i.e. :class:`~PauliSentence`) it should be
+            set to ``complex`` (default).
```

Comment:
Sorry, I'm coming into this without the context - but I've always struggled with the `dtype`-like arguments, with a feeling that these things should be automatically inferred where possible. Do we need the `dtype` in this case?

Reply:
Yes I think actually there is no necessity for it in the first place, I had a thinking error here 👍